tests/common.go

```diff
@@ -5131,3 +5131,140 @@ func GetPoolIDsFromVolName(volName string) ([]string, error) {
 	}
 	return poolUuids, err
 }
+
+// GetNodeByNodeName Get Node Details from Node Name Provided
+func GetNodeDetailsByNodeName(nodeName string) (*node.Node, error) {
+	Nodes, err := GetStorageNodes()
+	if err != nil {
+		return nil, err
+	}
+
+	for _, each := range Nodes {
+		if each.Name == nodeName {
+			return &each, nil
+		}
+	}
+	return nil, err
+}
+
+// GetPoolIOPriorityStatus Get Pool IO Priority status from PoolUUID
+func GetPoolIOPriorityStatus(poolUUID string) (string, error) {
+	var PoolIOPriority string
+	PoolIOPriority = ""
+	Pools, err := Inst().V.ListStoragePools(metav1.LabelSelector{})
+	log.FailOnError(err, fmt.Sprintf("Listing storage pools failed"))
+	for _, eachPool := range Pools {
+		if eachPool.Uuid == poolUUID {
+			PoolIOPriority = eachPool.Labels["iopriority"]
+			break
+		}
+	}
+	if PoolIOPriority != "" {
+		return "", errors.New(fmt.Sprintf("Failed to Get IO Priority for PoolUUID [%v]", poolUUID))
+	}
+	return PoolIOPriority, nil
+}
+
+// isNodeInMaintenanceMode returns true if Node in Maintenance
+func IsNodeInMaintenance(n node.Node) bool {
+	stNode, err := Inst().V.GetDriverNode(&n)
+	log.FailOnError(err, "Failed fetching Driver Details from Node [%v]", n)
```

Comment:
can you move this to node.go

Reply:
moved it to portworx.go as we depend on GetDriverNode value which is in portworx.go